pkgs/data/icons/la-capitaine-icon-theme/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "la-capitaine-icon-theme";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "keeferrourke";
+    repo = "la-capitaine-icon-theme";
+    rev = "v${version}";
+    sha256 = "0cm2scrcg5h45y56h822ywyvfzns1x4wf3gqq96cwb22dc7ny1g9";
+  };
+
+  postPatch = ''
+    rm configure
```

Comment:
Would be nice if this project had a proper Makefile, etc.

Looks like people had to do some interesting things to package the variants.
https://copr-dist-git.fedorainfracloud.org/cgit/tcg/themes/la-capitaine-icon-theme.git/tree/la-capitaine-icon-theme.spec#n69

Reply:
Is it Ok?
https://github.com/LinArcX/la-capitaine-icon-theme/tree/add-meson
If yes i can PR them to merge this and then polish __default.nix__.

Reply:
Looks like it would install a subdirectory `La-Capitaine` to the correct location but the organization in tree wouldn't work for that. There's also functionality within the configure script (variants) which is missing.

However you can use a custom install phase for this package  for now, and then pr them with your change. Hope I didn't give the impression that was necessary.